packages/js-lib/package.json

```diff
@@ -53,6 +53,7 @@
     "./string": "./dist/string/index.js",
     "./string/*.js": "./dist/string/*.js",
     "./types": "./dist/types.js",
+    "./typeFest": "./dist/typeFest.js",
```

Comment:
No, I don't think that it deserves its own "export". Instead, I'd add `export * from './typeFest.js'` at the end of `types.js`, so it becomes part of the same `@naturalcycles/js-lib/types` export